compiler/shared_ast/typing.ml

```diff
@@ -361,6 +368,26 @@ module Env = struct
         (A.ScopeName.Map.find scope_name t.scopes)
     in
     { t with scope_vars }
+
+  let rec dump ppf env =
+    let pp_sep = Format.pp_print_space in
+    Format.pp_open_vbox ppf 0;
+    (* Format.fprintf ppf "structs: @[<hov>%a@]@,"
```

Comment:
Dead code?

Reply:
Well technically this function is dead code, it shouldn't be used outside of debugging sessions — but pretty convenient nevertheless and we shouldn't need to rewrite it every time. The same applies to the commented code: while you generally wouldn't want all the typedecls listed here, I thought that having this available to be uncommented here could be helpful in debugging sessions where you might actually have use for it.

But well, since it's normally unused we may indeed prefer to uncomment to make sure the extended function continues to typecheck...